portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,21 +41,28 @@
     
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException { 
+      
       Query query = new Query("CommentData").addSort("timestamp", SortDirection.DESCENDING);
       DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
       PreparedQuery results = datastore.prepare(query); 
-
+      
+      
       List<CommentData> commentlog = new ArrayList<>();
       for (Entity entity : results.asIterable()) {
           long id = entity.getKey().getId();
           String userName = (String) entity.getProperty("userName");
-          String userComment = (String) entity.getProperty("userComment");
+          String Comment = (String) entity.getProperty("userComment");
           long timestamp = (long) entity.getProperty("timestamp");
+          String languageCode = request.getParameter("languageCode");
```

Comment:
nit: Because this isn't changing for each iteration of the for loop, it'd be more typical to move this line outside of the for loop.